### DISCUSSION TITLE
Feature Request: Encrypted Direct Messages

### DISCUSSION THREAD

**Discussion by njmulsqb** [mastodon/mastodon#25008] [Polls]
With Twitter rolling out encrypted DMs, I think it has become more important to have similar feature on Mastodon as well.

**Comment by shaedrich** [comment]
I think, the poll is not worded neutral. It's not that all Mastodon admins constantly read all our private conversations. So, wording it like one of these might be better:
* No, we are okay with the possibility of our private messages being read by a server admin
* No, we are okay with server admins potentially being able to read our private conversations

Also, it implies that with any encryption, server admins completely wouldn't be able to read private conversations, which is just not true either. Since they have to be decrypted somewhere and server admins controlling the server, they are also _potentialy_(!) able to tap into that. Not that they do, but it might technically be possible, depending on the implementation.

**Comment by shaedrich** [comment]
Duplicate of #21458
